docs/pages/Theming/Tokens/Basics.md

```diff
@@ -78,9 +78,11 @@ export const defaultAvatarTokens: TokenSettings<AvatarTokens, Theme> = (t: Theme
     ...
 ```
 
-### Font [in progress]
+### Font
 
-TO DO
+You can find global tokens related to fonts, including font families, sizes, and weights, in the `globalTokens.font` property.
+
+While you _could_ use these tokens directly in `Text` elements, this is not recommended. Instead, we recommend that you use the `variant` prop or the predefined JSX types as described in the [Typography](#typography) section, as these will pull in all the needed tokens automatically.
```

Comment:
I believe the Theme also has all the variants specified on it, which could be worth mentioning.

Reply:
Not a bad idea, although this really belongs in the Typography section.